apps/nextjs-website/src/app/page.tsx

```diff
@@ -41,12 +39,11 @@ const Home = async () => {
       <NotSsrWebinarHeaderBanner webinars={webinars} />
 
       <HeroSwiper
-        cards={homepage.hero.map((itemProp, index) => ({
+        cards={homepage.hero.map((itemProp) => ({
```

Comment:
Shouldn't each child in a list have a unique "key" prop?
```suggestion
        cards={homepage.hero.map((itemProp, index) => ({
```